### PR TITLE
Feature/report templates

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/model/Exam.java [hunk 1]
```diff
@@ -21,6 +23,7 @@
     private String completenessCode;
     private ScaleScore scaleScore;
     private List<ScaleScore> claimScaleScores;
+    private List<String> accommodationCodes;
```
Comment (esotrope) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/model/Exam.java]: preemptively adding accommodation codes for ica report view

File: common/src/main/resources/templates/fragments.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+    <head>
+    </head>
+    <body>
+
+        <div th:fragment="styles">
+            <link rel="stylesheet" href="sbac-ui-kit/dist/css/sbac-ui-kit.min.css">
```
Comment (esotrope) [inline common/src/main/resources/templates/fragments.html]: Playing around with the PDF generation made me realize we need to have a process for making the ui kit css/images/fonts available to wkhtmltopdf at processing time. I imagine this will mean remotely hosting the assets or adding a download step to our docker container so these assets can be accessed on the filesystem. Another option is embedding the css, images and fonts into the html template itself.
I plan on making a JIRA to work this out

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/report/PrintOptions.java [hunk 3]
```diff
@@ -1,16 +1,19 @@
 package org.opentestsystem.rdw.reporting.common.report;
 
 /**
- * Represents PDF generation specific options
+ * Represents printing options
  */
-public class PdfOptions {
+public class PrintOptions {
```
Comment (malaffoon) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/report/PrintOptions.java]: +1

File: common/src/main/resources/messages.properties [hunk 4]
```diff
@@ -69,260 +90,268 @@ report.ica.level.2.name=Standard Met (Level 3) {0}-{1}
...
-report.claim.3.grade.03.level.2=${student.firstName} demonstrates a thorough ability to clearly and precisely put together valid arguments to support his or her own mathematical thinking and to critique the reasoning of others.
-report.claim.3.grade.03.level.1=${student.firstName} demonstrates some ability to clearly and precisely put together valid arguments to support his or her own mathematical thinking and to critique the reasoning of others.
-report.claim.3.grade.03.level.0=${student.firstName} does not yet demonstrate the ability to put together valid arguments to support his or her own mathematical thinking or to critique the reasoning of others.
+report.overall-achievement.grade.03.subject.ELA={0} took a Smarter Balanced assessment based on the grade three California academic standards for English language arts/literacy and received an overall score that ranges from about 2000 to 3000. This overall score falls into one of the four achievement levels shown here. Sample test items may be included that exemplify the kinds of passages and questions that students at different levels of achievement would likely answer correctly.
```
Comment (malaffoon) [inline common/src/main/resources/messages.properties]: Just curious why the change from named to unnamed params?

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/report/iab/JdbcIabReportRepository.java [hunk 5]
```diff
@@ -41,21 +43,30 @@
...
+            final int subjectId) {
 
         final IabReport.Builder reportBuilder = IabReport.builder();
 
-        final Multimap<Assessment, Exam> assessments = TreeMultimap
-                .create(comparing(Assessment::getName), comparing(Exam::getDateTime).reversed());
+        final Multimap<Assessment, Exam> assessments = MultimapBuilder
+                .treeKeys(comparing(Assessment::getName))
+                .arrayListValues()
+                .build();
```
Comment (malaffoon) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/report/iab/JdbcIabReportRepository.java]: Is it one of those map-key-to-collection things?

File: common/src/main/resources/messages.properties (hunk 4 above)
Comment (esotrope) [inline common/src/main/resources/messages.properties]: java .properties message sources only allows nameless params it looks like

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/report/iab/JdbcIabReportRepository.java (hunk 5 above)
Comment (esotrope) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/report/iab/JdbcIabReportRepository.java]: I don't know - it's actually a bug with guava imo.
